apps/browser-tests/src/main.tsx

```diff
@@ -1,12 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
+import { WalletStandardProvider } from "@wallet-standard/react";
 
 import App from "./App";
 
 import "./style.css";
 
 ReactDOM.createRoot(document.getElementById("root")!).render(
   <React.StrictMode>
-    <App />
+    <WalletStandardProvider>
+      <App />
+    </WalletStandardProvider>
```

Comment:
Is it possible to remove this provider in ordit-sdk - are there base functions that wallet standard provides? ordit-sdk could possibly be used by js runtime without react, and this wouldn't work with ME wallets

we could still include it here for our test case but we should consider basic functions that standard provides

Reply:
there's @wallet-standard/core @danadi7 so you don't need to use react lib when getting the wallets list
https://github.com/wallet-standard/wallet-standard/blob/master/packages/example/wallets/src/window.ts#L14